msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,335 @@
+
+## Customize initialization
+
+You can initialize with your own adapter or customize after initialization.
+
+```js(ts)
+
+   // Create your own adapter
+   const adapter = new BotFrameworkAdapter(...);
+
+   // Customize your adapter, e.g., error handling
+   adapter.onTurnError = ...
+
+   const bot = new ConversationBot({
+       // use your own adapter
+       adapter: adapter;
+       ...
+   });
+
+   // Or, customize later
+   bot.adapter.onTurnError = ...
+
+```
+
+## Add command and response
+
+You can perform the following steps to add command and responses:
+
+<br>
+
+<details>
+
+<summary><b>1. Add a command definition in manifest</b></summary>
+
+You can edit the manifest template file `templates\appPackage\manifest.template.json` to include:
+
+* The command title that user types in the message compose area to trigger the command.
+* The description for the command.
+
+  :::image type="content" source="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png" alt-text="add a command definition in manifest code sample" lightbox="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png":::
+
+<br>
+</details>
+
+<details>
+
+<summary><b>2. Respond with an Adaptive Card</b></summary>
+
+You can build your response data in text format, or perform the following steps to use an Adaptive Card to apply rich content in Teams:
+
+* Prepare your Adaptive Card content in a JSON file such as `myCard.json` under the `bot/adaptiveCards` folder. Following is a sample Adaptive Card with JSON payload:
+
+  ```JSON
+        {
+            "type": "AdaptiveCard",
+            "body": [
+            {
+                "type": "TextBlock",
+                "size": "Medium",
+                "weight": "Bolder",
+                "text": "Your Hello World Bot is Running"
+            },
+            {
+                "type": "TextBlock",
+                "text": "Congratulations! Your hello world bot is running. Click the documentation below to learn more about Bots and the Teams Toolkit.",
+                "wrap": true
+            }
+            ],
+            "actions": [
+            {
+                "type": "Action.OpenUrl",
+                "title": "Bot Framework Docs",
+                "url": "https://docs.microsoft.com/en-us/azure/bot-service/?view=azure-bot-service-4.0"
+            },
+            {
+                "type": "Action.OpenUrl",
+                "title": "Teams Toolkit Docs",
+                "url": "https://aka.ms/teamsfx-docs"
+            }
+            ],
+            "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+            "version": "1.4"
+        }
+  ```
+
+* Import your card content into your code file where your command handler exists: `import myCard from "./adaptiveCards/myCard.json"`.
```

Comment:
Code snippet for JSON file import varies between TS and JS:
```typescript
import myCard from "./adaptiveCards/myCard.json";
```

```javascript
const myCard = require("./adaptiveCards/myCard.json");
```

Reply:
Should the file be changed from import myCard from "./adaptiveCards/myCard.json"; to const myCard = require("./adaptiveCards/myCard.json"); 

Can you elaborate what needs to be done?

Reply:
I mean the code snippet is different in TS and JS, we should provide both.

Reply:
This has been updated as the next suggested comment: 

Can we remove these parts from the 2. Respond with an Adaptive Card sections?

how to import card json
how to update the handleCommandReceived API
You can reference the latest Readme:
https://github.com/OfficeDev/TeamsFx/tree/dev/templates/bot/ts/command-and-response#step-2-respond-with-an-adaptive-card